src/pages/periodValidity.tsx

```diff
@@ -40,55 +94,9 @@ const PeriodValidity = ({ errors = [], csrfToken }: PeriodValidityProps): ReactE
-                                                errors.length > 0 ? 'govuk-input--error' : ''
-                                            } `}
-                                            id="period-end-calendar"
-                                            name="periodValid"
-                                            type="radio"
-                                            value="endOfCalendarDay"
-                                        />
-                                        <label
-                                            className="govuk-label govuk-radios__label govuk-label--s"
-                                            htmlFor="period-end-calendar"
-                                        >
-                                            At the end of a calendar day
-                                        </label>
-                                        <span className="govuk-hint govuk-radios__hint" id="period-end-calendar-hint">
-                                            For example, a ticket purchased at 3pm would be valid until midnight on its
-                                            day of expiry
-                                        </span>
-                                    </div>
-                                    <div className="govuk-radios__item">
-                                        <input
-                                            className={`govuk-radios__input ${
-                                                errors.length > 0 ? 'govuk-input--error' : ''
-                                            } `}
-                                            id="period-twenty-four-hours"
-                                            name="periodValid"
-                                            type="radio"
-                                            value="24hr"
-                                        />
-                                        <label
-                                            className="govuk-label govuk-radios__label govuk-label--s"
-                                            htmlFor="period-twenty-four-hours"
-                                        >
-                                            At the end of a 24 hour period from purchase
-                                        </label>
-                                        <span
-                                            className="govuk-hint govuk-radios__hint"
-                                            id="period-twenty-four-hours-hint"
-                                        >
-                                            For example, a ticket purchased at 3pm will be valid until 3pm on its day of
-                                            expiry
-                                        </span>
-                                    </div>
-                                </div>
-                            </FormElementWrapper>
+                            {fieldsets.map(fieldset => {
```

Comment:
Rather than have a `.map()` here, we could amend the `getFieldsets` method to return just the one fieldset item and pass just that into the `RadioConditionalInput` component?

Reply:
we do the same in other files as well such as

  {fieldsets.map(fieldset => {
                        return <RadioConditionalInput key={fieldset.heading.id} fieldset={fieldset} />;
                    })}